packages/tests/.solhintignore

```diff
@@ -1 +1,2 @@
-src/interfaces/
\ No newline at end of file
+src/interfaces/
+test/playground/
```

Comment:
do we want to commit the playground?

Reply:
I think it's fine for it to be in the ignore file, wdyt?